indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -695,9 +695,12 @@ public void doRun()
           for (File file : toMerge) {
             indexes.add(HadoopDruidIndexerConfig.INDEX_IO.loadIndex(file));
           }
+
+          log.info("starting merge of intermediate persisted segments...");
```

Comment:
this is only going to get printed once during batch ingestion and most of the time users are not running indexing with debug level. its not going to pollute the logs really.